attrs_parse: strip whitespace from end of key

spoore1 opened the pull request:
AD user get data was being returned with left over white space at the end of the keys.  Changing lstrip to strip in attrs_parse misc function to remove the whitespace at the end of the keys as well as the beginning.

pbrezina commented:
Can you add also unit test for this to https://github.com/SSSD/sssd-test-framework/blob/master/tests/test_misc.py? And more importantly 

```
tests/test_misc.py::test_attrs_parse__long_line[input1-expected1] FAILED [ 14%]
```

spoore1 commented:
Added test case and fixed the long line failure:

```
tests/test_misc.py::test_attrs_parse__long_line[input0-expected0] PASSED                                                                                                                                    [ 10%]
tests/test_misc.py::test_attrs_parse__long_line[input1-expected1] PASSED                                                                                                                                    [ 14%]
tests/test_misc.py::test_attrs_parse__long_line[input2-expected2] PASSED                                                                                                                                    [ 17%]
tests/test_misc.py::test_attrs_parse__strip_extra_white_space[input0-expected0] PASSED                                                                                                                      [ 21%]
```
